CONTRIBUTING.md

```diff
@@ -1,16 +1,518 @@
+   to a new section with a header similar to the following:
+
+   ```
+   ## [1.4.1](https://github.com/cyberark/conjur-google-cloud-launcher/releases/tag/v1.4.1) - 2020-03-18
+   ```
+
+1. Commit these changes - `Bump version to x.y.z` is an acceptable commit
+   message - and open a PR for review. Your PR should include updates to
+   `CHANGELOG.md`, and if there are any license updates, to `NOTICES.txt`.
+
+### Automated Push of Images to Google Cloud Storage by Jenkins Pipeline
+
+Note that when changes are merged into master via the steps above, the
+conjur-google-cloud-launcher Jenkins pipeline will automatically tag and
+push new versions of the Conjur Google Marketplace Application component
+images.
+
+### Add a git tag
+1. Once your changes have been reviewed and merged into master, tag the version
+   using e.g. `git tag -s v1.4.1`. Note this requires you to be  able to sign
+   releases. Consult the
+   [github documentation on signing commits](https://help.github.com/articles/signing-commits-with-gpg/)
+   on how to set this up. `vx.y.z` is an acceptable tag message.
+1. Push the tag: `git push vx.y.z` (or `git push origin vx.y.z` if you are working
+   from your local machine).
+
+### Build a release
+**Note:** Until the stable quality exercises have completed, the GitHub release
+should be officially marked as a `pre-release` (eg "non-production ready")
+1. From a **clean checkout of master** run `./bin/build_release` to generate
+   the release artifacts.
+1. Create a GitHub release from the tag, add a description by copying the CHANGELOG entries
+   from the version, and upload the release artifacts from `dist/goreleaser`
+   to the GitHub release. The following artifacts should be uploaded to the release:
+   - CHANGELOG.md
+   - NOTICES.txt
+   - LICENSE
+
+## Publishing to Google Marketplace
+
+Reference: https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/maintaining-solution
+
+### Prerequisites for Publishing to Google Marketplace 
+
+As decribed in the [Prerequisites for Publishing to Google Marketplace](#prerequisites-for-publishing-to-google-marketplace)
+above, publishing images to the Google Marketplace will require:
+* Access to the `conjur-cloud-launcher-onboard` Google Cloud Platform (GCP) project
+* Access to the `cyberark-public` GCP project
+
+### Check for Vulnerabilities in Images Pushed to GCR
+
+In a browser, navigate to the following GCR locations, and click on
+the `Vulnerabilies` button to check for vulnerabilies:
+
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/nginx:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/postgres:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/tester:1.4.1
+*https://gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:1.4.1
```

Comment:
Is this a styling `*` or a listing `*`? If it's the former, use `- ` instead and if it's styling, use `_<text>_`.